webui/react/src/e2e/tests/experimentList.spec.ts

```diff
@@ -439,4 +441,120 @@ test.describe('Experiment List', () => {
       expectPageNumber(null);
     });
   });
+  test.describe('Experiment List Multi-sort', () => {
+    const runScenarioAndValidation = (projectDetailsPage: ProjectDetails) => {
+      const multiSortMenu = projectDetailsPage.f_experimentList.tableActionBar.multiSortMenu;
+      const validateByColumn = async (
+        rows: ExperimentRow[],
+        colKey: string,
+        descending: boolean,
+      ) => {
+        const valuesToCompare = await Promise.all(
+          rows.map(async (r) => await (await r.getCellByColumnName(colKey)).pwLocator.innerText()),
```

Comment:
nit: you don't need await here because `innerText()` is already a promise:
```suggestion
          rows.map(async (r) => (await r.getCellByColumnName(colKey)).pwLocator.innerText()),
```